src/luarocks/core/cfg.lua

```diff
@@ -508,6 +508,22 @@ local function make_rocks_provided(lua_version, luajit_version)
       rocks_provided_3_0["tarantool"] = tarantool_version.."-1"
    end
 
+   if cfg.lua_interpreter == "tt" then
```

Comment:
Please add a comment describing why we should set the `tarantool` version this way when we use `tt` as interpreter.

Reply:
Added